docs/setup-cluster/slurm/slurm-known-issues.rst

```diff
@@ -381,6 +381,10 @@ Some constraints are due to differences in behavior between Docker and Singulari
  PBS Known Issues
 ******************
 
+-  If the ``Cluster`` tab on the DeterminedAI WebUI does not display the GPU information, there
+   could be an issue in the PBS configuration. Please refer to :ref:`PBS Requirements
+   <pbs-config-requirements>` to ensure PBS is configured properly.
+
```

Comment:
-  If the ``Cluster`` tab in the WebUI does not display the GPU information, there
   may be an issue with the PBS configuration. Visit :ref:`PBS Requirements
   <pbs-config-requirements>` to ensure PBS is properly configured.

Reply:
@jagadeesh545 avoid `DeterminedAI` as it is reserved for talking about the company itself. `Determined` is reserved for talking about the product. However, we can avoid it altogether. In the future we may also avoid `WebUI` but for now it's the most valid term.

Reply:
Sure, Tara. I updated as per your suggestions.